src/context/XDEFIProvider/types.ts

```diff
@@ -0,0 +1,27 @@
+import { HDWallet } from '@shapeshiftoss/hdwallet-core'
+
+import { AnyFunction } from '../../types/common'
+
+export interface InitialState {
+  ethereumWallet: HDWallet | null
+  xfiBitcoinProvider: IXfiBitcoinProvider | null
+  xfiLitecoinProvider: IXfiLitecoinProvider | null
+  isWalletLoading: boolean
+}
```

Comment:
The state interface that is provided by the `useXDEFIProvider` hook. 

The etereumWallet is the Wallet created when connecting via Shapeshift's interface. this is for easy access to the other wallets when connected to any other ethereum wallet.